src/authorizations/components/redesigned/ResourceAccordion.tsx

```diff
@@ -72,32 +86,40 @@ class ResourceAccordion extends Component<Props, State> {
             permissions={permissions[resource]}
             onToggleAll={this.handleToggleAll}
           />
-          <GetResources resources={[ResourceType[resourceName]]}>
-            {!isEmpty(permissions[resource].sublevelPermissions) &&
-              this.getAccordionBody(resourceName, resource)}
-          </GetResources>
+          {resourceName === 'Telegrafs' || resourceName === 'Buckets' ? (
+            <GetResources resources={[ResourceType[resourceName]]}>
+              {!isEmpty(permissions[resource].sublevelPermissions) &&
+                this.getAccordionBody(resourceName, resource)}
+            </GetResources>
+          ) : null}
         </Accordion>
       )
     })
   }
 
   getAccordionBody = (resourceName, resource) => {
     const {permissions} = this.state
-    return resourceName === 'Telegrafs' ? (
-      <ResourceAccordionBody
-        resourceName={resource}
-        permissions={permissions[resource].sublevelPermissions}
-        onToggle={this.handleIndividualToggle}
-        title="Individual Telegraf Configuration Names"
-      />
-    ) : (
-      <ResourceAccordionBody
-        resourceName={resource}
-        permissions={permissions[resource].sublevelPermissions}
-        onToggle={this.handleIndividualToggle}
-        title="Individual Bucket Names"
-      />
-    )
+    if (resourceName === 'Telegrafs') {
+      return (
+        <ResourceAccordionBody
+          resourceName={resource}
+          permissions={permissions[resource].sublevelPermissions}
+          onToggle={this.handleIndividualToggle}
+          title="Individual Telegraf Configuration Names"
+        />
+      )
+    } else if (resourceName === 'Buckets') {
```

Comment:
again, not a huge deal, but stylistically you can have less syntax by removing these `else` statements since each `if` statement `returns`